cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/service/http/DefaultHttpServiceResponder.java

```diff
@@ -67,8 +73,7 @@ public void sendJson(Object object) {
    */
   @Override
   public void sendJson(int status, Object object) {
-    responder.sendJson(HttpResponseStatus.valueOf(status), object);
-    emitMetrics(status);
+    sendString(status, GSON.toJson(object), Charsets.UTF_8);
```

Comment:
this would send the Json with a content-type of text/plain. Shouldn't it be application.json? In other words, should this be:
{code}
    send(status, Charsets.UTF_8.encode(GSON.toJson(object)), "application/json", null);
{code}